src/components/raycaster.js

```diff
@@ -143,9 +145,10 @@ module.exports.Component = registerComponent('raycaster', {
     });
 
     // Update intersectedEls.
-    intersectedEls = this.intersectedEls = intersections.map(function getEl (intersection) {
-      return intersection.object.el;
-    });
+    intersectedEls.length = intersections.length;
```

Comment:
The intent was to avoid creating new arrays, not about disposing elements. The elements will still be there, but at least we reuse the same array.